src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.nuspec

```diff
@@ -35,7 +29,15 @@
     <tags>TraceEvent EventSource Microsoft ETW Event Tracing for Windows</tags>
 
     <dependencies>
-      <dependency id="System.Runtime.CompilerServices.Unsafe" version="4.5.2" />
```

Comment:
Can you clarify what this change does?

Reply:
It's covered here, in the steps for "Manually edited nuspec file":
https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128#solution

Reply:
Cool, thanks for the pointer.